class.jetpack-jitm.php

```diff
@@ -135,9 +150,40 @@ function manage_pi_msg() {
 		if ( current_user_can( 'jetpack_manage_modules' ) ) {
```

Comment:
Instead of wrapping all of this logic in a conditional, we could fail out early (even as early as in the beginning of `prepare_jitms()`)